### DISCUSSION TITLE
High Availability and Failover with Operator

### DISCUSSION THREAD

**Discussion by agentp-1337** [arkmq-org/arkmq-org-broker-operator#523] [Q&A] [answered]
Based on the following documentation https://activemq.apache.org/components/artemis/documentation/latest/ha, is it possible to configure the a colocated or primary/backup setup? I could not find any corresponding CRD attribute which would enable such a config. Many thx for your support.

**Comment by brusdev** [comment, marked as answer]
The HA on Kubernetes/OpenShift is provided by the platform itself and the ArtemisCloud Operator supports message migration if you scale down your cluster. There is no support for  colocated or primary/backup setup.

**Reply by agentp-1337** [reply]
Many thx for your fast response.
